assets/js/modules/analytics/common/account-select.test.js

```diff
@@ -114,9 +114,11 @@ describe( 'AccountSelect', () => {
 	it( 'should pre-select the property and profile IDs when changed', () => {
 		const { accounts, properties, profiles } = fixtures.accountsPropertiesProfiles;
 		const { getByText, container, registry } = render( <AccountSelect />, { setupRegistry } );
+		const propertyID = fixtures.accountsPropertiesProfiles.properties[ 0 ].id;
+		const accountID = fixtures.accountsPropertiesProfiles.properties[ 0 ].accountId;
```

Comment:
Nit-picking, but this can be simplified to reference `properties[ 0 ]` since that is already retrieved 2 lines above.

Reply:
Fixed